src/main/kotlin/com/yapp/brake/common/event/EventPayloadHandler.kt

```diff
@@ -0,0 +1,7 @@
+package com.yapp.brake.common.event
+
+interface EventPayloadHandler<T : EventPayload> {
```

Comment:
```suggestion
interface EventPayloadHandler<in T : EventPayload> {
```

이렇게 하면 EventRelay에서 강제 캐스트보다 컴파일 타임에 안정성을 확보할 수 있지 않을까요 ??
다른 이유가 있다면 공유해주시면 감사하겠습니다 🙇‍♀️